CSS/index.css

```diff
@@ -35,6 +35,121 @@
   margin: 0;
   font-size: 48px;
 }
+.menu-button {
+  position: absolute;
```

Comment:
I would play around with this because if the size gets smaller, the menu is covering the header text.